core-war/src/main/java/org/silverpeas/web/pdc/servlets/PdcSearchRequestRouter.java

```diff
@@ -1377,29 +1051,17 @@ private void setDefaultDataToNavigation(HttpServletRequest request,
    */
   private List<Integer> searchAllSilverContentId(PdcSearchSessionController pdcSC,
       QueryParameters searchParameters) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00112)